app/components/fathom.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
+import { load, trackPageview } from 'fathom-client';
+import { usePathname, useSearchParams } from 'next/navigation';
+import { useEffect } from 'react';
+
+export default function Fathom() {
+  const pathname = usePathname();
+  const searchParams = useSearchParams();
```

Comment:
Please add a space after between blocks - e.g. `searchParams` and `useEffect` here

Reply:
Done, not sure why Prettier didn't pick this up.